proto/ibc/core/channel/v2/packet.proto

```diff
@@ -39,13 +39,7 @@ message Payload {
 
 // Acknowledgement contains a list of all ack results associated with a single packet.
 message Acknowledgement {
-  repeated AcknowledgementResult acknowledgement_results = 1 [(gogoproto.nullable) = false];
-}
-
-// AcknowledgementResult of identified acknowledgement. Correlating IBC application name to opaque ack bytes.
-message AcknowledgementResult {
```

Comment:
Unnecessary struct since the acknowledgment side only ever uses the Acknowledgement bytes inside the `RecvPacketResult`.

The `status` in RecvPacketResult is **only** used on the receive side. Thus we do not need to transmit it over the wire back to the sending chain. Instead we can just package each individual opaque app acknowledgement into the Acknowledgment struct and send it over.

Moreover, the commitment isn't using the status so it isn't even secure to use it on the sending side (which we don't)